daemon/swaps/40swap.go

```diff
@@ -135,6 +135,44 @@ func (f *Client) GetSwapOut(ctx context.Context, swapId string) (*SwapOutRespons
 	return &swapOutResponse, nil
 }
 
+func (f *Client) GetClaimPSBT(ctx context.Context, swapId, address string) (*GetClaimPSBTResponse, error) {
+	params := api.SwapOutControllerGetClaimPsbtParams{
+		Address: address,
+	}
+	response, err := f.client.SwapOutControllerGetClaimPsbt(ctx, swapId, &params)
+	if err != nil {
+		return nil, err
+	}
+
+	if response.StatusCode >= 400 {
+		return nil, fmt.Errorf("failed to get claim PSBT: %d - %s", response.StatusCode, response.Status)
+	}
+
+	var getClaimPSBTResponse GetClaimPSBTResponse
+	err = json.NewDecoder(response.Body).Decode(&getClaimPSBTResponse)
+	if err != nil {
+		return nil, err
+	}
+
+	return &getClaimPSBTResponse, nil
+}
+
+func (f *Client) PostClaim(ctx context.Context, swapId, tx string) (*PostClaimResponse, error) {
```

Comment:
if we don't need anything we can just remove the param
```suggestion
func (f *Client) PostClaim(ctx context.Context, swapId, tx string)  error {
```

Reply:
done, good point